roles/server/defaults/main.yml

```diff
@@ -35,9 +35,11 @@ domain: '{{ inventory_hostname }}'
 hardware:
 - cpu: 'host'
 - vcpus: '2'
-- memory: '1024'
+- memory: '2048'
 - virt_type: 'kvm'
-- graphics: 'spice,listen=socket'
+#- graphics: 'spice,listen=socket'
+- graphics: 'vnc,listen=0.0.0.0'
```

Comment:
These are role defaults. Please do not change those. If they do not fit your use case, please define/overwrite variables in your Ansible inventory. Please have a look at [Ansible's variable precedence rules](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable).